patroni/postgresql.py

```diff
@@ -1224,8 +1224,8 @@ def controldata(self):
         if self._version_file_exists() and self.state != 'creating replica':
             try:
                 data = subprocess.check_output([self._pgcommand('pg_controldata'), self._data_dir],
-                                               env={'LANG': 'C', 'LC_ALL': 'C', 'PATH': os.environ['PATH'],
-                                               'SYSTEMROOT': os.environ['SYSTEMROOT']})
+                                               env={'LANG': 'C', 'LC_ALL': 'C', 'PATH': os.environ.get('PATH'),
+                                               'SYSTEMROOT': os.environ.get('SYSTEMROOT')})
```

Comment:
The value of the environment variable must be a string, but it this case the value of `SYSTEMROOT` would be `None`, therefore it fails on travis.